tracim/tracim/model/data.py

```diff
@@ -54,11 +51,18 @@ class Workspace(DeclarativeBase):
     label   = Column(Unicode(1024), unique=False, nullable=False, default='')
     description = Column(Text(), unique=False, nullable=False, default='')
 
-    created = Column(DateTime, unique=False, nullable=False)
-    updated = Column(DateTime, unique=False, nullable=False)
+    created = Column(DateTime, unique=False, nullable=False, default=datetime.utcnow)
+    updated = Column(DateTime, unique=False, nullable=False, default=datetime.utcnow)
```

Comment:
Add your comment in the source code
`Default value datetime.utcnow, see: http://stackoverflow.com/a/13370382/801924 (or http://pastebin.com/VLyWktUn)`